src/macrofier/engine.mjs

```diff
@@ -1070,7 +1070,7 @@ function insertMethodMacros(template, methodObj, json, templates, examples = {})
   }
   const method = {
     name: methodObj.name,
-    params: methodObj.params.map(p => p.name).join(', '),
+    params: methodObj.params.map(p => p.name).join(''),
```

Comment:
This property is for providing a list of parameters, why'd you remove the comma? we want:

`param1, param2`

not 

`param1param2`